packages/app-project/src/shared/components/CollectionsModal/components/SelectCollection/SelectCollection.js

```diff
@@ -60,7 +70,7 @@ function SelectCollection ({
           dropHeight='medium'
           dropProps={dropProps}
           id='collectionsSearch'
-          labelKey='display_name'
+          labelKey={collectionLabel}
```

Comment:
Label key should be the name of an object property in the `option` object, not the property value. 

https://v2.grommet.io/select#labelKey

Reply:
Can you elaborate on this statement? According to that link, labelKey can be a function that returns a string. I'm confused about what's wrong here.

Reply:
This should be a string that names an option property, like `display_name`, or a function that returns a dynamic value, like `collectionLabel`.

Reply:
Oh wait, is `collectionLabel` a function? My mistake.